.stylelintrc.json

```diff
@@ -21,6 +21,17 @@
     "scss/at-import-no-partial-leading-underscore": true,
     "scss/at-import-partial-extension": "never",
 
+    "order/order": [
```

Comment:
There are more ordering rules that could be added here but this seemed like a good set of rules without updating a lot of the codebase.